docs/contributing/ways_to_contribute.rst

```diff
@@ -10,7 +10,8 @@ little bit helps, and credit will always be given.
 Talk to us
 ----------
 
-* Get support in our `Community Forums <http://community.learningequality.org/>`__.
+* Get product support in our `Community Forums <http://community.learningequality.org/>`__.
+* Get development contributions support in `Kolibri GitHub Discussions <https://github.com/learningequality/kolibri/discussions>`__.
 * Email us at info@learningequality.org
 * Visit the ``#kolibri`` room on Freenode IRC
```

Comment:
We should probably remove this, I don't think this room is still active.

Reply:
Okay